plugins/woocommerce/tests/e2e-pw/tests/merchant/create-variable-product.spec.js

```diff
@@ -14,7 +14,7 @@ const defaultAttributes = [ 'val2', 'val1', 'val2' ];
 const stockAmount = '100';
 const lowStockAmount = '10';
 
-test.describe.serial( 'Add New Variable Product Page', () => {
```

Comment:
We don't need to specify serial here because the tests within a suite will run sequentially

Reply:
I would still recommend retaining `serial` since the 2nd test, _"can set the variation attributes, bulk edit variations"_, relies on the variable product that was created by the first test, _"can create product, attributes and variations"_.

Without `serial`, when the 2nd test became flaky in the first try, the retry will start with the 2nd test without rerunning the first test. The 2nd test will then fail because it will have no variable product to use, even though it wasn't already flaky by then.

<img width="1026" alt="KDmkE8KuMe" src="https://user-images.githubusercontent.com/4509348/227227015-a1aa1e8b-91a1-459e-b234-3a563d0c64be.png">

But with `serial`, the 2nd test will have a chance to pass since the first test was retried too.
<img width="1147" alt="EZy9FG7NvS" src="https://user-images.githubusercontent.com/4509348/227227196-b221a7d6-6c19-4219-a8d7-015a56007f2d.png">

Reply:
I refactored and combined the tests into 2 tests -- because we shouldn't have been doing that (copied the logic over from the Puppeteer suite).  So they definitely don't need to run serially anymore.